AspFluent[] fluents
AspFunction[] atoms
bool satisfied
